src/Symfony/Component/Cache/DependencyInjection/CachePoolPass.php

```diff
@@ -169,6 +170,7 @@ public function process(ContainerBuilder $container)
             }
 
             $allPools[$name] = new Reference($id, $container::IGNORE_ON_UNINITIALIZED_REFERENCE);
+            $allPoolsReference[$name] = new Reference($id);
```

Comment:
This breaks the current behavior, where unused cache pools (from a service wiring pov) are removed from the container.
Which use case made you add this line?